File: packages/cli/test/fixtures/multilang/sample.py
from os.path import join, dirname
import json

class UserService:
    def __init__(self, db):
        self.db = db

    def find_by_id(self, user_id):
        return self.db.get(user_id)

    def create(self, name, email):
        validate_email(email)
        return {"id": generate_id(), "name": name, "email": email}

def validate_email(email):
    if "@" not in email:
        raise ValueError("Invalid email")

def generate_id():
    import uuid
    return str(uuid.uuid4())

File: packages/cli/test/fixtures/multilang/sample.rb
class UserService
  def initialize(db)
    @db = db
  end

  def find_by_id(user_id)
    @db.get(user_id)
  end

  def create(name, email)
    validate_email(email)
    { id: generate_id, name: name, email: email }
  end
end

module Helpers
  def self.format_name(name)
    name.strip.downcase
  end
end

def validate_email(email)
  raise "Invalid email" unless email.include?("@")
end

def generate_id
  SecureRandom.uuid
end
